contents/swift/welcome-to-swift.md

```diff
@@ -173,7 +216,7 @@ You can think of Structs as a way to create instances that have their own unique
 
```

Comment:
I feel that this section on structs is not really convincing me how it is a benefit. Perhaps you could include 1-2 sentences on in what kind of situations using a value-typed struct could prove to be more useful/more appropriate than using a reference-type class?